tests/Models/Multiplayer/RoomTest.php

```diff
@@ -137,4 +170,18 @@ public function testCannotStartPlayedItem()
         $this->expectException(InvariantException::class);
         (new Room())->startGame($user, $params);
     }
+
+    public function startGameDurationDataProvider()
+    {
+        return [
+            '2 weeks' => [20160, true, null],
```

Comment:
all those numbers are rather hard to read (and update). Put them in variables with original calculation instead...